resources/views/PageDesign/Partials/Header/Breadcrumb.twig

```diff
@@ -91,6 +91,7 @@
         {
             "@context":"http://schema.org/",
             "@type":"BreadcrumbList",
+            "name":"Breadcrumb Navigation",
```

Comment:
Whats the name of the list used for? We already have names for each breadcrumb item... Google does not set this on its example as well:
https://developers.google.com/search/docs/data-types/breadcrumb?hl=en

Reply:
When Testing the page with Google Rich-Test-Tool, the navigation is not named "unnamed element" no more.
There might be no need to add this to Ceres standard.

Reply:
If it is not mandatorily required, please remove the static name or replace it by an optional translation key the user can change by himself. If you do so, please add an empty translation key to make sure the generated output does not change since the user changes the translation.